plugin/codex-agent-identity/plugin.go

```
@@ -143,39 +152,26 @@ func handleMethod(method string, request []byte) ([]byte, error) {
-				Label:       labelFromMetadata(req.Metadata, req.AuthID),
-				StorageJSON: req.StorageJSON,
-				Metadata:    req.Metadata,
-				Attributes:  req.Attributes,
-			},
-			NextRefreshAfter: time.Now().Add(24 * time.Hour).UTC(),
+			Auth:             refreshed,
+			NextRefreshAfter: refreshed.NextRefreshAfter,
 		})
 	case pluginabi.MethodAuthLoginStart:
-		return okEnvelope(pluginapi.AuthLoginStartResponse{Provider: "codex", ExpiresAt: time.Now().UTC()})
+		return nil, errors.New("Codex Agent Identity login is managed through the plugin management page; native Codex OAuth login is unchanged")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)